### PR TITLE
Refactor configuration 

### LINKED ISSUE #234: Profiling is impossible on the script
## What would you like to be added

Currently, due to the usage of `__main__` to import the `config` object from ArgParse, it is impossible to use external scripts that can instrument the main script (because it changes the entrypoint script).

Making it impossible to use: `cProfile`, `memory_profiler` for instance.

A solution is to create another file `conf.py` and add a global `config` object in it which is initialized from the `kube-hunter.py` script and every other parts of the script can depend on the `conf` module rather than `__main__` (which violates PEP8 BTW).

## Why is this needed

Profiling the script helps developers to debug performance/memory issues, which is critical for a security script.

### PR DESCRIPTION
**Pull request by iYehuda**
## Description
Moved configuration into a separate package and break dependencies of `__main__` module.
This way we can run external tools on kube-hunter (e.g., pytest, profiling, etc.).
runtest.py may now be deprecated and removed.

## Contribution Guidelines
Please Read through the [Contribution Guidelines](https://github.com/aquasecurity/kube-hunter/blob/master/CONTRIBUTING.md).

## Fixed Issues
Fixes #234 

## Contribution checklist
 - [x] I have read the Contributing Guidelines.
 - [x] The commits refer to an active issue in the repository.
 
## Notes
This PR was initiated by @RaitoBezarius, thank you for your work!

### CODE REVIEW

File: runtest.py [hunk 1]
```diff
@@ -1,27 +1,12 @@
...
-parser.add_argument('--remote', nargs='+', metavar="HOST", default=list(), help="one or more remote ip/dns to hunt")
-parser.add_argument('--active', action="store_true", help="enables active hunting")
-parser.add_argument('--log', type=str, metavar="LOGLEVEL", default='INFO', help="set log level, options are: debug, info, warn, none")
-parser.add_argument('--report', type=str, default='plain', help="set report type, options are: plain, yaml")
-parser.add_argument('--statistics', action="store_true", help="set hunting statistics")
-
-config = parser.parse_args()
+#!/usr/bin/env python3
 
+import argparse
```
Comment (itaysk) [inline runtest.py]: shouldn't this import conf instead of argparse?

File: kube-hunter.py [hunk 2]
```diff
@@ -1,63 +1,55 @@
...
-
-import plugins
-
-config = parser.parse_args()
-
-try:
-    loglevel = getattr(logging, config.log.upper())
-except:
-    pass
+# TODO: move log level parsing to conf module
```
Comment (itaysk) [inline kube-hunter.py]: should we create a new issue to track the conf refactor and add these TODOs there? I understand you don't want to block this PR on all other little refactoring but we should be able to better track it

Comment (iYehuda) [reply kube-hunter.py]: I believe it is a good idea

File: runtest.py (hunk 1 above)
Comment (iYehuda) [reply runtest.py]: It should be deleted entirely, but considering this file is made irrelevant by #272 we can ignore that.

File: kube-hunter.py (hunk 2 above)
Comment (itaysk) [reply kube-hunter.py]: created https://github.com/aquasecurity/kube-hunter/issues/289. please remove the comments before merging

**Pull request merged**